include/zephyr/net/net_ip.h

```diff
@@ -50,6 +50,9 @@ extern "C" {
 #define PF_LOCAL        6          /**< Inter-process communication   */
 #define PF_UNIX         PF_LOCAL   /**< Inter-process communication   */
 
+/** First ID available for downstream protocols. Must be last/biggest numbered PF_* entry. */
```

Comment:
Have you really tested that this works as expected because there is only three bits reserved for the family value in net_context.h https://github.com/zephyrproject-rtos/zephyr/blob/9f60075d14a283602a3580b6b97dd4686b53a28e/include/zephyr/net/net_context.h#L60

There is also checks when setting the family in https://github.com/zephyrproject-rtos/zephyr/blob/9f60075d14a283602a3580b6b97dd4686b53a28e/include/zephyr/net/net_context.h#L561

Reply:
No, not yet tested. Not with this number. Will take it out of draft mode once actually done.

Reply:
No worries, I just noticed the draft and took a look and commented about potential issues in it.